src/Discovery/DiscoveryDropdownTagViewer.tsx

```diff
@@ -113,16 +116,46 @@ const DiscoveryDropdownTagViewer: React.FunctionComponent<DiscoveryTagViewerProp
 
           return (
             <Col
-              className='discovery-header__tag-group'
+            className='discovery-header__tag-group'
               key={category.name}
               xs={24}
               sm={24}
               md={12}
               lg={12}
               xl={12}
               xxl={12}
+
             >
-              { tags }
+              <table>
+                <tr>
+                  <td>
+                  <Col>
+                  <div style={{ display: category.tooltip != null ? 'block' : 'none' }}>
```

Comment:
don't use CSS to hide a mounted element. you should be doing something like this https://github.com/uc-cdis/data-portal/blob/master/src/Discovery/DiscoveryDetails.tsx#L234-L267 to not mounting an element in the first place